src/kili/services/asset_import/__init__.py

```diff
@@ -18,7 +18,7 @@
     "IMAGE": ImageDataImporter,
     "TEXT": TextDataImporter,
     "VIDEO": VideoDataImporter,
-    "FRAMES": VideoDataImporter,
+    "VIDEO_LEGACY": VideoDataImporter,
```

Comment:
I admit that when coding the new importer, I didn't even think to the VIDEO_LEGACY input_type. I would have to see if it works with the VideoDataImporter!